apps/atomy-q/API/tests/Feature/ProviderQuoteExtractionTest.php

```
@@ -0,0 +1,184 @@
+    {
+        config()->set('atomy.ai.mode', 'provider');
+        config()->set('atomy.ai.endpoints.document.uri', 'https://openrouter.example.test/chat/completions');
+        config()->set('atomy.ai.endpoints.document.model_id', 'baidu/qianfan-ocr-fast:free');
+        config()->set('atomy.ai.endpoints.document.parser_plugin', 'file-parser');
+        config()->set('atomy.ai.endpoints.document.pdf_engine', 'mistral-ocr');
+        config()->set('atomy.ai.endpoints.document.health_url', 'https://openrouter.example.test/chat/completions/health');
+        $transport = new class implements ProviderAiTransportInterface {
+            /** @var array<string, mixed> */
+            public array $payload = [];
+
+            public function invoke(string $endpointGroup, array $payload): array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$endpointGroup'. (undefined)

(UnusedFormalParameter)